hooks/Validators/README.md

```diff
@@ -14,15 +14,15 @@ This package also builds on various PSRs (PHP Standards Recommendations) such as
 
 ## Installation
 
-```
-$ composer require open-feature/validators-hook   // installs the latest version
+```sh
+composer require open-feature/validators-hook
 ```
 
 ## Usage
 
 The following validator hook constructs are available, but more are being worked on over time:
 
-- `RegexpValidatorHoook`
+- `RegexpValidatorHook`
 
 
 ```php
```

Comment:
It may also be worth adding a section that discourages registering a validation hook at the global or client level. While it's possible, it should only be done if the consequences are well understood.

Reply:
Things like registering OpenTelemetry or DDTrace are global, but I've also implemented autoloading for those so no explicit action is required to just work with those specifically.

Additional hooks could be logging hooks which would likely be global/client.

The comments do indicate the implications of such a registration, without diving too deep into it